gremlin-server/src/main/java/org/apache/tinkerpop/gremlin/server/GremlinServer.java

```diff
@@ -99,11 +114,16 @@ public GremlinServer(final Settings settings) {
     public GremlinServer(final ServerGremlinExecutor<EventLoopGroup> serverGremlinExecutor) {
         this.serverGremlinExecutor = serverGremlinExecutor;
         this.settings = serverGremlinExecutor.getSettings();
+        this.isLinuxOS = settings.useEpollEventLoop && SystemUtils.IS_OS_LINUX;
```

Comment:
I think that it would be worth doing a `logger.warn()` after this line if someone tries to "useEpoll" but are not on linux.  I'd further add a `logger.info()` to indicate the event loop ultimately used.